extensions/amp-sidebar/validator-amp-sidebar.protoascii

```diff
@@ -44,7 +44,6 @@ tags: {  # <amp-sidebar> not in amp-story
   tag_name: "AMP-SIDEBAR"
   # There is an alternate spec for amp-sidebar in amp-story.
   disallowed_ancestor: "AMP-STORY"
-  mandatory_parent: "BODY"
```

Comment:
Should the `mandatory_parent: "AMP-STORY"` be removed from the stanza below as well, since `<amp-story>` is essentially the `<body>` in STAMPs?

Reply:
In my opinion we should keep it until a use case comes up